From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
Date: Sun, 5 Jun 2016 18:47:56 +0300
Subject: [ARCHEOLOGY] Power LED patch for H3 SPL

> X-Git-Archeology: - Revision 55032f5bf7464b68528137a3e71e42de6d0355c5: https://github.com/armbian/build/commit/55032f5bf7464b68528137a3e71e42de6d0355c5
> X-Git-Archeology:   Date: Sun, 05 Jun 2016 18:47:56 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: Power LED patch for H3 SPL
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2abd603134083a909818a029e06d33c216ce7ee7: https://github.com/armbian/build/commit/2abd603134083a909818a029e06d33c216ce7ee7
> X-Git-Archeology:   Date: Mon, 28 Nov 2016 11:45:33 +0100
> X-Git-Archeology:   From: Thomas Kaiser <github@kaiser-edv.de>
> X-Git-Archeology:   Subject: Add scaling_available_freqs sysfs node, enable led based on DT
> X-Git-Archeology:
> X-Git-Archeology: - Revision 82dbcf0a5d8c23034c042b9df74a393b3a01461d: https://github.com/armbian/build/commit/82dbcf0a5d8c23034c042b9df74a393b3a01461d
> X-Git-Archeology:   Date: Tue, 29 Nov 2016 05:23:06 +0100
> X-Git-Archeology:   From: Thomas Kaiser <github@kaiser-edv.de>
> X-Git-Archeology:   Subject: Revert 'enable led based on DT' since needs massive testing
> X-Git-Archeology:
> X-Git-Archeology: - Revision dddf22832564dda0102ea07bcc56fbbefae3b050: https://github.com/armbian/build/commit/dddf22832564dda0102ea07bcc56fbbefae3b050
> X-Git-Archeology:   Date: Tue, 29 Nov 2016 14:27:57 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: Merge and rename u-boot patch directories
> X-Git-Archeology:
> X-Git-Archeology: - Revision f6a09def35e647a5442e0f92f399485be29f19a9: https://github.com/armbian/build/commit/f6a09def35e647a5442e0f92f399485be29f19a9
> X-Git-Archeology:   Date: Thu, 10 Nov 2022 21:49:36 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Moving patches to per board and removing obsolete (#4409)
> X-Git-Archeology:
---
 board/sunxi/board.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/board/sunxi/board.c b/board/sunxi/board.c
index 28f702bc29..bee01a2c3f 100644
--- a/board/sunxi/board.c
+++ b/board/sunxi/board.c
@@ -632,6 +632,11 @@ void sunxi_board_init(void)
 		status_led_init();
 #endif
 
+#ifdef CONFIG_MACH_SUN8I_H3
+	/* turn on power LED (PL10) on H3 boards */
+	gpio_direction_output(SUNXI_GPL(10), 1);
+#endif
+
 #ifdef CONFIG_SY8106A_POWER
 	power_failed = sy8106a_set_vout1(CONFIG_SY8106A_VOUT1_VOLT);
 #endif
-- 
Armbian

